src/twisted/web/http_headers.py

```diff
@@ -247,7 +247,7 @@ def getRawHeaders(
             return default
 
         if isinstance(name, str):
-            return [v if isinstance(v, str) else v.decode("utf8") for v in values]
```

Comment:
`v` is always `bytes`; it comes from `self._rawHeaders`, to which we only ever write `bytes` in for both names and values.